src/internal/testing/TestScheduler.ts

```diff
@@ -131,6 +132,30 @@ export class TestScheduler extends VirtualTimeScheduler {
     };
   }
 
+  expectSubscriptionsTo(observable: ColdObservable<any>|HotObservable<any>): ({ toBe: subscriptionLogsToBeFn }) {
+    const actual: any[] = [];
+    const flushTest: FlushableTest = { actual, ready: false };
+
+    this.schedule(() => {
+      observable.subscriptions.map(log => {
+        actual.push(log);
+      });
+    }, 0);
+
+    return {
+      toBe: (marbles: string | string[]) => {
+        const marblesArray: string[] = (typeof marbles === 'string') ? [marbles] : marbles;
+        flushTest.ready = true;
+        flushTest.expected = marblesArray.map(marbles =>
+          TestScheduler.parseMarblesAsSubscriptions(marbles, this.runMode)
+        );
+      }
+    };
+  }
```

Comment:
I don't see how this performs an expectation. Can a test that deliberately fails - due to an invalid subscriptions expectation - be added to show that this works and isn't just a noop?

Reply:
Added. Good idea.